examples/FunctionAppIsolatedMode/ShowBetaFeature.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.FeatureManagement;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.FunctionAppIsolatedMode
+{
+    public class ShowBetaFeature
+    {
+        private readonly IFeatureManagerSnapshot _featureManagerSnapshot;
+        private readonly ILogger _logger;
+
+        public ShowBetaFeature(IFeatureManagerSnapshot featureManagerSnapshot, ILoggerFactory loggerFactory)
+        {
+            _featureManagerSnapshot = featureManagerSnapshot;
+            _logger = loggerFactory.CreateLogger<ShowBetaFeature>();
+        }
+
+        [Function("ShowBetaFeature")]
+        public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req)
+        {
+            _logger.LogInformation("C# HTTP trigger function processed a request.");
+
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            response.Headers.Add("Content-Type", "text/plain; charset=utf-8");
+
+            // Read feature flag
+            string featureName = "Beta";
+            bool featureEnalbed = await _featureManagerSnapshot.IsEnabledAsync(featureName);
```

Comment:
`featureEnabled`. Also do you mind putting new lines between the statements here.

Reply:
I had to remove the sample app because the build pipeline doesn't have the required tools to build function apps in isolated mode.